lib/rucio/client/commands/bin_legacy/rucio.py

```diff
@@ -1993,15 +1993,18 @@ def add_lifetime_exception(args, client, logger, console, spinner):
     dids_list = []
     containers = []
     datasets = []
-    error_types = ['Total DIDs',
-                   'DID not submitted because it is a file',
-                   'DID that are containers and were resolved',
-                   'DID not submitted because it is not part of the lifetime campaign',
-                   'DID successfully submitted including the one from containers resolved']
+    error_type_mapping = {
+        "total_dids": "Total DIDs",
+        "files_ignored": "DID not submitted because it is a file",
+        "containers_resolved": "DID that are containers and were resolved",
+        "not_in_lifetime_model": "DID not submitted because it is not part of the lifetime campaign",
+        "successfully_submitted": "DID successfully submitted including the one from containers resolved"
+    }
```

Comment:
I don't see why we would assign those strings to the values of this dict - they are essentially comments on what each counter represents, so they should just be comments.

Reply:
Aren't these dictionary values being printed along with the count? We're using the summary dictionary keys to map to proper descriptions, so we no longer rely on positional indexes (as in the previous implementation) when displaying the information.
(https://github.com/rucio/rucio/blob/dbb9c6faf4b02e48de2983fb31dc01e90e93eabc/lib/rucio/client/commands/bin_legacy/rucio.py#L2073)
Like For example, the output now looks like this:
```
Total DIDs                                                                                      1282
DID not submitted because it is a file                                                            0
DID that are containers and were resolved                                                        1282
```